upload/admin/view/template/extension/shipping/zasilkovna.twig

```diff
@@ -67,6 +67,23 @@
 									</select>
 								</div>
 							</div>
+							<div class="form-group">
+								<label class="col-sm-2 control-label"  for="input-pricing-by">{{ entry_pricing_by }}</label>
+								<div class="col-sm-10">
+									<select name="shipping_zasilkovna_pricing_by" id="input-pricing-by" class="form-control">
+										{% for pricing_by_type in pricing_by_types %}
```

Comment:
V tomto případě, bych naplnění selectu kde jsou 2 možnosti neřešil iterací pole.   Tato možnost je vhodná v případech, kdy možnosti v selectu načítáš z databáze.  Bude to přehlednější.